### ISSUE TITLE
"path" false positive

### ISSUE THREAD

**Issue by fleetwire** [depcheck/depcheck#751]
`"path": "^0.12.7"` dependency shows up as unused. I use it in two files via `const path = require('path');`

Files where `path` is used:`webpack.config.js` & `vue.config.js` in the root folder.

```json
{
  "dependencies": ["@nesk/puphpeteer", "bootstrap", "dotenv", "path", "vue-style-loader"],
  "devDependencies": [
    "@mdi/font",
    "sass",
    "sass-loader",
    "string-replace-loader",
    "vue-loader",
    "webpack-visualizer-plugin"
  ],
  "missing": {
    "eslint-plugin-vue": ["/Users/chad/Sites/fleetwire/.eslintrc.json"],
    "eslint-config-airbnb-base": ["/Users/chad/Sites/fleetwire/.eslintrc.json"],
    "lodash": [
      "/Users/chad/Sites/fleetwire/resources/js/tenant/fwtent.js",
      "/Users/chad/Sites/fleetwire/resources/js/tenant/Services/button.service.js",
      "/Users/chad/Sites/fleetwire/resources/js/tenant/Services/card.service.js",
      "/Users/chad/Sites/fleetwire/resources/js/tenant/Services/datepicker.service.js",
      "/Users/chad/Sites/fleetwire/resources/js/tenant/Services/listing.service.js",
      "/Users/chad/Sites/fleetwire/resources/js/tenant/Services/search.service.js",
      "/Users/chad/Sites/fleetwire/resources/js/tenant/Services/util.service.js"
    ]
  }
}
```

**Comment by znarf** [comment]
`path` is a core node module, you usually don't have to put in your dependencies.
